server/models.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable */
+// Written for Postgres
+const Image = require('../database/index_postgres');
+
+module.exports = {
+  restaurants: {
+    get: (params, callback) => {
+      Image.findAll({
+        where: {
+          userid: parseInt(params)
+        }
+      }).then(function (result) {
```

Comment:
in this current implementation, if there is no error and you pass in a callback you will need to pass in the result as second param e.g. `function(null, result){}`. This depends however on how you decide to handle errors in `controllers.js`